Sources/Paywall/Models/Paywall/PresentationCondition.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Yusuf Tör on 05/07/2022.
+//
+
+import Foundation
+
+enum PresentationCondition: String, Decodable {
+  case always = "ALWAYS"
+  case checkPrimarySubscription = "CHECK_PRIMARY_SUBSCRIPTION"
```

Comment:
Just changed this to `CHECK_USER_SUBSCRIPTION`

Reply:
Thanks, changed my end too, I'll merge this in now